index.js

```diff
@@ -426,18 +435,10 @@ function Argv (processArgs, cwd) {
       }
     }
 
+    // we must run completions first, a user might
+    // want to complete the --help or --version option.
     Object.keys(argv).forEach(function (key) {
```

Comment:
I'm curious about this style of coding (`Object.keys(argv).forEach(func))` when you're only checking for the presence of a single key).  I would find this more straightforward:

```js
if (completionOpt in argv) {
  // do whatever
}
```

Are you intentionally guarding against `completionOpt` being in the `Object` prototype?